sound/soc/sof/ipc.c

```diff
@@ -1061,8 +1015,8 @@ struct snd_sof_ipc *snd_sof_ipc_init(struct snd_sof_dev *sdev)
 	ipc->ops = &ipc3_ops;
 
 	/* check for mandatory ops */
-	if (!ipc->ops->pcm || !ipc->ops->tplg || !ipc->ops->tplg->widget ||
-	    !ipc->ops->tplg->control) {
+	if (!ipc->ops->get_reply || !ipc->ops->pcm || !ipc->ops->tplg ||
```

Comment:
see earlier comment, why is get_reply mandatory and not the others? tx_msg, rx_msg and set_get_data are missing. is this intentional, and if yes a comment would help.

Reply:
That's a good question, they should all be mandatory.